### ISSUE TITLE
length gets imported as length$0

### ISSUE THREAD

**Issue by P0lip** [stoplightio/spectral#1783]
**Describe the bug**

What the title says.

**To Reproduce**

1. Given the following YAML ruleset
```yaml
rules:
  rule:
    given: $
    then:
      function: truthy
  valid-length:
    given: $
    then:
      function: length
```
2. Run migration
3. Get the following output

```
const {
  truthy: truthy,
  length$0: length$0,
} = require("@stoplight/spectral-functions");
module.exports = {
  rules: {
    rule: {
      given: "$",
      then: {
        function: truthy,
      },
    },
    "valid-length": {
      given: "$",
      then: {
        function: length$0,
      },
    },
  },
};
```

**Expected behavior**

```js
const {
  truthy: truthy,
  length: length$0,
} = require("@stoplight/spectral-functions");
module.exports = {
  rules: {
    rule: {
      given: "$",
      then: {
        function: truthy,
      },
    },
    "valid-length": {
      given: "$",
      then: {
        function: length$0,
      },
    },
  },
};
```

Related to https://github.com/stoplightio/platform-internal/issues/7541
